sdk/monitor/Azure.Monitor.Ingestion/tests/MonitorIngestionLiveTest.cs

```diff
@@ -159,5 +141,47 @@ public async Task InvalidInputFromObjectAsJsonNoBatchingNoGzipAsync()
             Assert.AreEqual(413, response.Value.Errors.FirstOrDefault().Error.Code);
             Assert.AreEqual(10000, response.Value.Errors.FirstOrDefault().FailedLogs.Count());
         }
+
+        [AsyncOnly]
+        [Test]
+        [LiveOnly]
+        public async Task ConcurrencyMultiThread()
+        {
+            var policy = new ConcurrencyCounterPolicy();
+            LogsIngestionClient client = CreateClient(policy);
+            LogsIngestionClient.SingleUploadThreshold = 1000; // make batch size smaller for Uploads for test recording size
+
+            // Make the request
+            UploadLogsOptions options = new UploadLogsOptions();
+            options.MaxConcurrency = 10;
+            System.Runtime.CompilerServices.ConfiguredTaskAwaitable<Response<UploadLogsResult>> tasks;
+            using (Recording.DisableRecording())
```

Comment:
Hmm, why do we need this? The test is marked as LiveOnly so it should not be doing any recording.